source/includes/api/tables/progress-response.rst

```diff
@@ -81,3 +81,17 @@
      - If an error occurred, gives a detailed description of the error.
        This field is omitted when the call to the endpoint is successful
 
+   * - ``mongosyncID``
+     - string
+     - The identifier string for the ``mongosync`` instance.
+       To configure this value, see the :setting:`id` setting.
+
+   * - ``coordinatorID``
+     - string
+     - The identifier string for the coordinator instance.  To change this value,
+       see the :setting:`id` setting on the coordinator instance of ``mongosync``.
```

Comment:
I am not sure what the behavior would be if the `id` of the coordinator instance is changed. Assuming that a restart is needed, would the resulting mongosync still be the coordinator instance?

Please check in #mongosync-impl, but in general, I think we shouldn't tell users about the possibility of changing `coordinatorID`. Changing `mongosyncID` is still permitted, and the side-effect of that concerning which instance is the coordinator should be documented there.

Reply:
This isn't exactly possible / how it works. There's nothing tying a mongosync kernel process / PID to the coordinator role.

The only persistent notion of a coordinator role is what the value is of the `coordinator` field in the destination metadata (specifically the global state document). This is derived from one of the mongosyncs' `--id` CLI value and stays constant throughout a migration and "cannot" be changed. If you wanted to change this, you'd have to manually change the `coordinator` field in a database that's called `mongosync_reserved_for_internal_use`, but we broadly say that any manual adjustments to such metadata will lead to undefined behavior.

If, say, a user were running two mongosyncs (each on a different machine) and wanted switch which mongosync acts as the coordinator because it's on a preferred machine, they'd have to shut down both processes and swap their `--id` values on the command line. This basically switches their "roles".

Still, I agree that we probably shouldn't be advertising this to users right now. We'd probably need separate, more formal documentation around this, since it's not totally obvious how this works.

Reply:
Fixed